examples/object_detection_3d/README.md

```diff
@@ -0,0 +1,105 @@
+# Example Integration: KITTI 3D Object Detection
+
+This example integration uses the image and LIDAR data from the KITTI dataset to
+demonstrate how to test 3D Object Detection on Kolena.
+
+## Setup
+
+This project uses [Poetry](https://python-poetry.org/) for packaging and Python dependency management. To get started,
+install project dependencies from [`pyproject.toml`](./pyproject.toml) by running:
+
+```shell
+poetry update && poetry install
+```
+
+## Usage
+
+This example integration will use pre-trained models provided by [MMDetection3D](https://github.com/open-mmlab/mmdetection3d/blob/main/docs/en/model_zoo.md) library. Please follow the [Installation](https://mmdetection3d.readthedocs.io/en/latest/get_started.html#installation) and set up your environment.
+
+### Data preparation
+
+We will use the same data directory structure as detailed in [KITTI dataset preparation guide](https://mmdetection3d.readthedocs.io/en/latest/advanced_guides/datasets/kitti.html), replacing the root directory in the guide with current project directory.
+
+### Test the model
+
+After setting up the KITTI dataset as instructed in the guide,
+
+In this example, we will use [SECOND](https://github.com/open-mmlab/mmdetection3d/tree/main/configs/second) baseline. You can choose other suitable models with the same process.
+
+```
+mim download mmdet3d --config second_hv_secfpn_8xb6-80e_kitti-3d-3class --dest checkpoints
```

Comment:
q: we don't need to use `mim` under the poetry ctx?

Reply:
The command is to download pre-trained model provided by mmdetection3d/openmmlab

Reply:
i guess it's installed by `openmim` package in the poetry ctx. `poetry run mim --help` can call it properly, unless we installed it globally like via conda, i suspect we can directly call it

Reply:
Would it be easier to just include inferences as a file in `s3://kolena-public-datasets`, and leave the actual downloading/running of models to the user?

Reply:
I am uncertain if sharing inference results on KITTI dataset qualifies as dataset usage, which has non-commercial usage clause. Therefore doing this in a safer way.